articles/healthcare-apis/iot/iot-connector-overview.md

```diff
@@ -17,9 +17,12 @@ ms.author: jasteppe
 
 ## Overview
 
-IoT connector is an optional service of the Azure Healthcare APIs that provides the capability to ingest message data from Internet of Medical Things (IoMT) devices and wearables. 
+IoT connector is an optional service of the Azure Healthcare APIs designed to ingest health data from multiple and disparate Internet of Medical Things (IoMT) devices and persisting the health data in a FHIR server.
```

Comment:
We should use FHIR service vs. FHIR server to stay consistent with marketing and what it is called in the UI.

Reply:
Yes, I will change from server to service.